extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -177,8 +176,22 @@ class AmpAccordion extends AMP.BaseElement {
   }
 
   /**
-   * Handles key presses on an accordion expand/collapse its content or
-   * move focus to previous/next header.
+   * Handles clicks on an accordion header to expand/collapse its content.
+   */
+  clickHandler_(event) {
+    // Need to support clicks on any children of the header except
+    // for on links, which should not have their default behavior
+    // overidden.
+    const target = dev().assertElement(event.target);
+    if (target.tagName != 'A') {
```

Comment:
this may not work for image links (`<a href><amp-img>...`). I think we need to check all the ancestors of target up to the header element. `ancestorElements(child, predicate)` should help find it.